react-app/src/native/components/register/ChooseLeague.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types' 
+import { connect } from 'react-redux'
+import {Picker, View, Icon } from 'native-base'
+import { fetchLeague } from '../../../actions/leagues' //Gets the leagues from the web.
+import DateTimeHelpers from '../../../utils/datetimehelpers'
+
+//Expects a sport id given as 1-4 under the 'sport' props tag
+
+class PickLeagues extends React.Component { 
+  
+  //I make a server call to get all the leagues. In the props (sport=(1-4)) the correct leagues for that sport appear
+  static propTypes = {
+    seasons: PropTypes.object.isRequired,
+    isLoading: PropTypes.bool.isRequired,
+    leagues: PropTypes.object.isRequired,
+    getLeague: PropTypes.func.isRequired 
+  }
+  
+  constructor(props) {
+    super(props)
+    state = {
+      counter:[1]
+    }
+  }
+
+  render() {
+    
+    const {
+      loading,
+      seasons
+    } = this.props
+
+
+    if (seasons == null || seasons == undefined) console.log("Error loading seasons");
+
+    return (
+      <View>
+        <Picker
+          placeholder="League"
+          note={false}
+          mode="dropdown"
+          iosIcon={<Icon name="arrow-down" />}
+          selectedValue = {this.props.league}
+          onValueChange={(elem) => {
+            console.log("Elem = " + elem)
+            if (elem == 1 || elem == undefined || elem == null) return;
+            
+            let leagueName
+            seasons[this.props.sport][0].leagues.map(curLeague => {
+              if (curLeague.id == elem) {   
+                leagueName =
+                  curLeague.name +
+                  ' - ' +
+                  DateTimeHelpers.getDayString(curLeague.dayNumber)
+                return
+              }
+            })
+            this.props.update(this.props.index, leagueName)
+          }}
+          style = {{
```

Comment:
Don't add style specifically to one picker in the app, all pickers should look and work the same so either add it globally or remove it.